packages/mds-utils/utils.ts

```diff
@@ -251,7 +251,7 @@ function pointInGeometry(pt: [number, number], shape: Geometry): boolean {
     return pointInPolygon(pt, shape)
   }
   if (shape.type === 'Point') {
-    if (pointInPolygon(pt, circleToPolygon(shape.coordinates, RADIUS, NUMBER_OF_EDGES))) {
+    if (pointInPolygon(pt, circleToPolygon(shape.coordinates.slice(0, 2), RADIUS, NUMBER_OF_EDGES))) {
```

Comment:
I think this is due to the dependency upgrades you made, @drtyh2o identified this when bumping dependencies previously and opened a github issue against the `circleToPolygon` package. We decided to keep the code the same, and just use the previous version, should probably revert that one dep upgrade.